cmd/kcp/kcp.go

```diff
@@ -70,8 +72,8 @@ func main() {
 		},
 	}
 	cfg = server.BindOptions(server.DefaultConfig(), startCmd.Flags())
-	cmd.AddCommand(startCmd)
-	if err := cmd.Execute(); err != nil {
-		fmt.Fprintf(os.Stderr, "error: %v\n", err)
-	}
+
+	kcpCmd.AddCommand(startCmd)
+	code := cli.Run(kcpCmd)
```

Comment:
I pulled over the cli.Run helper, but not the help/usage func code, because I think that generally assumes there are no subcommands. We can undo this if y'all don't like it...